data/json/uncraft/ammo/9mm.json

```diff
@@ -3,28 +3,28 @@
     "result": "9mm",
     "copy-from": "ammo_cartridge",
     "type": "uncraft",
-    "using": [ [ "ammo_9mm", 1 ], [ "ammo_bullet", 1 ] ],
+    "using": [ [ "ammo_9mm", 1 ], [ "ammo_bullet", 2 ] ],
```

Comment:
That was a leftover from when I added `ammo_bullet_fmj`. I left it in on removing it due to noticing that the full metal jacket recipe uses half as much lead as the other rounds. I can undo this change if recommended.